### ISSUE TITLE
Main ways to avoid overfitting?

### ISSUE THREAD

**Issue by ImBadAtNames2019** [ExponentialML/Text-To-Video-Finetuning#44]

**Comment by ExponentialML** [comment]
The rule of thumb is:

- Diverse dataset. Less is more. If you have a lot of data, but there are anomalies, training will suffer.
- Keep your learning rate consistent. The general numbers I see are 2e-6, 5e-6, 1e-5.
- Increase the weight decay of the AdamW optimizer.
- Use dropout if *not* doing LoRA training.
